src/pages/[platform]/build-a-backend/auth/connect-your-frontend/sign-in/index.mdx

```diff
@@ -840,7 +844,10 @@ await autoSignIn();
 ```
 
 </InlineFilter>
-<InlineFilter filters={['react-native']}>
+<InlineFilter filters={['react-native']}
+>
+
+To sign in using an external identity provider such as Google, use the `signInWithRedirect` function.
```

Comment:
```suggestion
```
I think this can be removed from this section. React Native `signInWithRedirect` is documented above with the other JS libs.